source/MaterialXCore/Document.h

```diff
@@ -672,6 +672,16 @@ class Document : public GraphElement
     /// @return True if the document passes all tests, false otherwise.
     bool validate(string* message = nullptr) const override;
 
+    /// @}
+    /// @name Versioning
+    /// @{
+
+    /// Convert Parameter Elements to Input Elements
+    bool convertParametersToInputs();
```

Comment:
Should this be called convertParametersToUniformInputs? Or should the other function be called convertUniformsToParameters for consistency?

Reply:
When 1.38 is done the uniforms may or may not be uniform so leaving it as is. The other one method will be removed on 1.38 release.